bears/natural_language/SpellCheckBear.py

```diff
@@ -10,7 +10,14 @@ class SpellCheckBear:
     """
     Lints files to check for incorrect spellings using ``scspell``.
 
-    See <https://pypi.python.org/pypi/scspell3k> for more information.
+    scspell is a spell checker for source code.
+    When applied to a code written in most popular programming languages
+    while using most typical naming conventions, this algorithm will
+    usually catch many errors without an annoying false positive rate.
+
+    In an effort to catch more spelling errors, scspell is able to
+    check each file against a set of dictionary words selected
+    specifically for that file.
```

Comment:
Added. Sorry, I missed out the "add-the-link-to-its-package" part in the previous comment. My bad.